fastai/vision/data.py

```diff
@@ -11,8 +11,18 @@
 
 # Cell
 @delegates(subplots)
-def get_grid(n, nrows=None, ncols=None, add_vert=0, figsize=None, double=False, title=None, return_fig=False,
-             flatten=True, **kwargs):
+def get_grid(
+    n, # Number of axes in the returned grid
+    nrows=None, # Number of rows in the returned grid, defaulting to `int(math.sqrt(n))`
+    ncols=None, # Number of columns in the returned grid, defaulting to `ceil(n/rows)`
+    add_vert=0,
```

Comment:
It's not used in the function I brought it up with @muellerzr and he said to leave it here and submit a pr in fastai to remove it. I figured nothing to document since its not used.

Reply:
And yes I will add return typing

Reply:
@muellerzr I was going to do a pr on fastai just to delete the unused `add_vert` argument. So I went to fork the fastai repo. It took me strait to the the fork of this (doc sprint) repo, which I already had. I figured I shouldn't use this fork since it would include a lot more changes than just the deletion of add_vert.

Reply:
Interesting. It might need some special rebasing magic or something.